src/stores/GlobalStore.js

```diff
@@ -105,44 +163,90 @@ export default class GlobalStore {
     });
   }
 
+  /**
+   * Handles an onApproveSuccess subscription message.
+   * @param {object} data Subscription message payload.
+   */
+  onApproveSuccess = (data) => {
+    if (data.error) {
+      console.error(data.error.message); // eslint-disable-line no-console
+      return;
+    }
+
+    if (!this.localWallet) {
+      // TODO: execute follow up tx
+    }
+  }
+
+  /**
+   * Subscribe to onApproveSuccess subscription.
+   * Receives messages of approve transactions that were successful.
+   * This is used so the UI knows to execute a follow-up tx.
+   */
+  subscribeOnApproveSuccess = () => {
```

Comment:
what is follow-up tx?

Reply:
Follow up txs are approve -> set result (follow-up tx), approve -> vote, approve -> create event. Actually I did not end up using this subscribe. I will remove it.

Reply:
Actually I might use this for something in the near future.